espresso/espresso.py

```diff
@@ -1516,11 +1517,11 @@ def write_input(self, inputname='pw.inp', calculation=None,
             if value is not None:
                 print('  {0:s}={1:s},'.format(attr, num2str(value)), file=finp)
 
-        sys_str_attrs = ['exxdiv_treatment']
+        sys_str_attrs = ['exxdiv_treatment','vdw_corr']
         for attr in sys_str_attrs:
             value = getattr(self, attr)
             if value is not None:
-                print('  {0:s}={1:s},'.format(attr, value), file=finp)
+                print('  {0:s}=\'{1:s}\','.format(attr, value), file=finp)
```

Comment:
how about updating this line to ` print("  {0:s}='{1:s}',".format(attr, value), file=finp)`

Reply:
I am okay  with both.  :-)